### PR TITLE
docs: add GEOGRAPHY data type code samples

### PR DESCRIPTION
**Pull request by tswast**
These are added to a separate directory in order to isolate the GeoJSON
and WKT dependencies from the other code samples.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Towards internal 171804665

### CODE REVIEW

File: tests/system.py [hunk 1]
```diff
@@ -2414,9 +2414,8 @@ def test_querying_data_w_timeout(self):
 
         query_job = Config.CLIENT.query(
             """
-            SELECT name, SUM(number) AS total_people
-            FROM `bigquery-public-data.usa_names.usa_1910_current`
-            GROUP BY name
+            SELECT COUNT(*)
```
Comment (tswast) [inline tests/system.py:2417]: Not really associated with this sample, but the system test flaked, so I figured I'd swap out this query with one from go/bq-long-queries

Comment (shollyman) [reply tests/system.py:2417]: Sounds good.

File: samples/geography/noxfile.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+from __future__ import print_function
+
+import os
+from pathlib import Path
+import sys
+
+import nox
+
+
+# WARNING - WARNING - WARNING - WARNING - WARNING
```
Comment (shollyman) [inline samples/geography/noxfile.py:24]: This file is generated?

**Review by shollyman — APPROVED** [review]
I was today years old when I found out that streaming inserts will accept either WKT or geojson as the field contents.

File: samples/geography/noxfile.py (hunk 2 above)
Comment (tswast) [reply samples/geography/noxfile.py:24]: Yerp. Copied from here: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_samples/noxfile.py.j2

**Pull request merged**